src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -34,31 +35,32 @@ private enum WellKnownAttributeFlags
             IsCallerFilePath = 0x1 << 5,
             IsCallerLineNumber = 0x1 << 6,
             IsCallerMemberName = 0x1 << 7,
+            HasInterpolatedStringArgumentAttributeError = 0x1 << 8,
         }
 
         private struct PackedFlags
         {
             // Layout:
-            // |...|fffffffff|n|rr|cccccccc|vvvvvvvv|
+            // |..|fffffffff|n|rr|ccccccccc|vvvvvvvvv|
```

Comment:
Rather than creating a new structure, I think the appropriate thing to do will be to move `_bits` to a long at that point. It'll take a bit of effort when merging, but I'd much rather do that than try and create a separate side-structure for just this bit.

Reply:
Talked with Aleksey offline, I misread his comment. I'll do this.